test/kixi/integration/base.clj

```diff
@@ -0,0 +1,125 @@
+(ns kixi.integration.base
+  (:require [byte-streams :as bs]
+            [clojure.test :refer :all   ;:exclude [deftest]
+             ]
+            [cheshire.core :as json]
+            [kixi.repl :as repl]
+            [clj-http.client :as client]
+            [clojure.java.io :as io]
+            [digest :as d])
+  (:import [java.io 
+            File
+            FileNotFoundException]))
+
+(defmacro is-submap
+  [expected actual]
+  `(try
+     (let [act# ~actual
+           exp# ~expected
+           [only-in-ex# only-in-ac# shared#] (clojure.data/diff exp# act#)]
+       (if only-in-ex#
+         (clojure.test/do-report {:type :fail
+                                  :message "Missing expected elements."
+                                  :expected only-in-ex# :actual act#})
+         (clojure.test/do-report {:type :pass
+                                  :message "Matched"
+                                  :expected exp# :actual act#})))
+     (catch Throwable t#
+       (clojure.test/do-report {:type :error :message "Exception diffing"
+                   :expected nil :actual t#}))))
+
+(defn cycle-system-fixture
+  [all-tests]
+  (repl/start)
+  (all-tests)
+  (repl/stop))
+
+(defn uuid
+  []
+  (str (java.util.UUID/randomUUID)))
+
+(defmacro deftest-broken
+  [name & everything]
+  `(clojure.test/deftest ~(vary-meta name assoc :integration true) ~@everything))
+
+(defn service-url
+  []
+  (or (System/getenv "SERVICE_URL") "localhost:8080"))
+
+(defn parse-json
```

Comment:
I think @acron0 is right on this one unless you are guarding against swapping json implementations.